tests/functional/amo_sign_up.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern!object',
+  'intern/chai!assert',
+  'tests/functional/lib/helpers'
+], function (registerSuite, assert, FunctionalHelpers) {
+
+  var MIGRATE_PARAMS = 'state=state&migration=amo&scope=scope';
```

Comment:
`scope=scope` is going to fail as soon as #3469 lands, can you use `scope=profile` so it simulates the AMO migration a bit better? That'll also set us up well for when we add the permissions screen. We should also pass a fake email address in using the `email` query parameter. You can just make up an email.